napari_allencell_annotator/widgets/annotation_widget.py

```diff
@@ -52,6 +52,8 @@ def add_existing_item(self, name: str, key_info: Key | ComboKey):
             item.fill_vals_check(name, key_info.get_default_value())
         elif annot_type == "list":
             item.fill_vals_list(name, key_info.get_default_value(), key_info.get_options())
+        else:
```

Comment:
elif

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/6bfbf5e42452477f6bdbed152dcf5f16c8a90a38